src/TSTransformer/classes/TransformState.ts

```diff
@@ -213,7 +213,8 @@ export class TransformState {
 				for (let i = 1; i < this.runtimeLibRbxPath.length; i++) {
 					expression = luau.create(luau.SyntaxKind.MethodCallExpression, {
 						expression,
-						name: "FindFirstChild",
+						// Lune does not support :WaitForChild()
+						name: process.env.npm_lifecycle_event !== "test-compile" ? "WaitForChild" : "FindFirstChild",
```

Comment:
This would also affect `run-in-roblox`, making that testing suite slightly less accurate. Is there a better (but easy) solution to _only_ trigger on lune testing?

Reply:
Added a test-roblox-compile script so we can avoid doing this when running run-in-roblox